rust/worker/benches/limit.rs

```diff
@@ -3,41 +3,52 @@ use chroma_types::{Chunk, SignedRoaringBitmap};
 use criterion::Criterion;
 use criterion::{criterion_group, criterion_main};
 use worker::execution::operator::Operator;
+use worker::execution::operators::fetch_segment::FetchSegmentOutput;
 use worker::execution::operators::limit::{LimitInput, LimitOperator};
-use worker::log::test::{add_generator_0, LogGenerator};
+use worker::log::test::{upsert_generator, LogGenerator};
 use worker::segment::test::TestSegment;
 
-const LIMIT: usize = 100;
+const FETCH: usize = 100;
```

Comment:
why this rename?

Reply:
To match the language of current query operator. Will update this once the language of the queryplan/queryoperator is updated.